python/lsst/ip/isr/calibType.py

```diff
@@ -261,11 +270,44 @@ def updateMetadata(self, camera=None, detector=None, filterName=None,
         self._metadata["DET_SER"] = self._detectorSerial if self._detectorSerial else None
         self._metadata["FILTER"] = self._filter if self._filter else None
         self._metadata["CALIB_ID"] = self._calibId if self._calibId else None
+        self._metadata["SEQFILE"] = self._seqfile if self._seqfile else None
+        self._metadata["SEQNAME"] = self._seqname if self._seqname else None
+        self._metadata["SEQCKSUM"] = self._seqcksum if self._seqcksum else None
         self._metadata["CALIBCLS"] = get_full_type_name(self)
 
         mdSupplemental.update(kwargs)
         mdOriginal.update(mdSupplemental)
 
+    def updateMetadataFromExposures(self, exposures):
+        """Extract and unify metadata information.
+
+        Parameters
+        ----------
+        exposures : `list`
+            Exposures or other calibrations to scan.
+        """
+        # This list of keywords is the set of header entries that
+        # should be checked and propagated.  Not having an entry is
+        # not a failure, as they may not be defined for the exposures
+        # being used.
+        keywords = ["SEQNAME", "SEQFILE", "SEQCKSUM", "ODP", "AP0_RC"]
+        metadata = {}
+
+        for exp in exposures:
+            try:
+                expMeta = exp.getMetadata()
+            except AttributeError:
+                continue
```

Comment:
Out of interest, if calibs have this and exposures have this, what do you expect to not have a metadata item? And I take it you want them all to compare correctly then without log warning or anything?

Reply:
Interesting... Well I'll leave it to you then, but if it's easy it might be worth seeing what breaks if you remove that and at least popping a comment in there for future-you.